### PR TITLE
quick update for dropwizard 1.0.0-rc1 compatibility

### PR DESCRIPTION
**Pull request by spudone**

### CODE REVIEW

**Comment by spudone** [comment]
Much love to dropwizard for mixing and matching java8 / guava Optionals...

**Comment by spudone** [comment]
Dropwizard 1.0.0 now requires Java 8; the CI failures were on the builders that attempted to use Java 7.

File: src/main/java/io/federecio/dropwizard/swagger/ConfigurationHelper.java [hunk 1]
```diff
@@ -60,7 +62,10 @@ public String getJerseyRootPath() {
                     .getJerseyRootPath();
         }
 
-        return stripUrlSlashes(rootPath);
+        if (rootPath.equals(Optional.absent()))
```
Comment (jplock) [inline src/main/java/io/federecio/dropwizard/swagger/ConfigurationHelper.java]: This should probably be `if (!rootPath.isPresent()) {`

Comment (spudone) [inline src/main/java/io/federecio/dropwizard/swagger/ConfigurationHelper.java]: Actually, probably should just drop the if statement and do:
`return (rootPath.or("/");`

**Comment by jplock** [comment]
Would you mind also removing oraclejdk7 and openjdk7 from the .travis.yml file and setting the `jdk.version` version to 1.8?

**Comment by spudone** [comment]
I had already updated jdk.version in pom.xml; didn't spot that .travis.yml before -- good catch.

**Comment by jplock** [comment]
Thanks!  I'll publish it later tonight.

**Pull request merged**